core/src/main/scala/org/apache/spark/deploy/worker/ExecutorRunner.scala

```diff
@@ -51,6 +51,19 @@ private[spark] class ExecutorRunner(
     var state: ExecutorState.Value)
   extends Logging {
 
+  // Grab all the SSL settings from the worker configuration
+  private val workerSecurityProps =
```

Comment:
This may not be needed if you update `SparkConf.isExecutorStartupConf`. See `SparkDeploySchedulerBackend.scala`, where it's used.

(Separately, it would probably be good for Yarn to use that same call instead of having it's own separate whitelist...)